go/ct/spc/specification.go

```diff
@@ -1652,10 +1672,36 @@ func getAllRules() []Rule {
 		},
 	})...)
 
-	// --- CALL, STATICCALL and DELEGATECALL ---
+	// --- CALL, CALLCODE, STATICCALL and DELEGATECALL ---
 
 	rules = append(rules, getRulesForAllCallTypes()...)
 
+	rules = append(rules, Rule{
+		Name: "call_not_enough_gas_for_new_account",
+		Parameter: []Parameter{
+			GasParameter{},
+			AddressParameter{},
+			ValueParameter{},
+			MemoryOffsetParameter{},
+			SizeParameter{},
+			MemoryOffsetParameter{},
+			SizeParameter{},
+		},
+		Condition: And(
+			Eq(Status(), st.Running),
+			Eq(Op(Pc()), vm.CALL),
```

Comment:
Because comment says `// --- CALL, CALLCODE, STATICCALL and DELEGATECALL ---` I would have expected this rule to be applied to the 4 calls. I can see that this a little of a problem because of semantics. 

The CT does not know that they are all using the same code, therefore we cannot assume that calling it for call makes it correct for the other 3 calls. 
How difficult would be to generate 4 rules from this code?

Reply:
this particular rules is only for `call` not all types of calls. This is one of the reasons why I decided to make it its own rule instead of trying to add it to `getRulesForAllCallTypes`. It is under `// --- CALL, CALLCODE, STATICCALL and DELEGATECALL ---` because it is a specialized rule of the more generic `getRulesForAllCallTypes` even if it applies to a single type of call